http/src/client/builder.rs

```diff
@@ -82,13 +84,13 @@ impl ClientBuilder {
         self
     }
 
-    /// Set the reqwest client to use.
+    /// Set a pre-configured reqwest client builder to build off of
```

Comment:
The period should stay.
```suggestion
    /// Set a pre-configured reqwest client builder to build off of.
```